apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php

```diff
@@ -287,19 +287,6 @@ function load_mailerlite() {
 }
 add_action( 'plugins_loaded', __NAMESPACE__ . '\load_mailerlite' );
 
-/**
- * Load WPCOM block editor nav sidebar.
- */
-function load_wpcom_block_editor_sidebar() {
-	if (
-		( defined( 'WPCOM_BLOCK_EDITOR_SIDEBAR' ) && WPCOM_BLOCK_EDITOR_SIDEBAR ) ||
-		apply_filters( 'a8c_enable_nav_sidebar', false )
-	) {
-		require_once __DIR__ . '/wpcom-block-editor-nav-sidebar/class-wpcom-block-editor-nav-sidebar.php';
```

Comment:
I don't think we will be able to deploy this PR if we delete the `required_once` AND the required files. We need to split into multiple PRs and deployments. 🤔

Reply:
Good catch 🙂
We will probably do the clean-up for all the features we've moved off ETK in one PR. I imagine we remove the require_once declarations first, then the files. Thoughts @fushar @arthur791004?

Reply:
Thanks! Yes, I just got a warning about the removal of a file in my sandbox. 

> We will probably do the clean-up for all the features we've moved off ETK in one PR

This is a smart approach!
As for this PR (and https://github.com/Automattic/wp-calypso/pull/92583), I think we can merge it first because of documenting their context of deprecation.

Reply:
I guess it's fine as the feature is disabled so the `require_once` won't be run anymore